pages/mintKamon/index.tsx

```diff
@@ -0,0 +1,36 @@
+import type { NextPage } from 'next'
+import { useAccount } from 'wagmi'
+import { Button, Text, Heading } from '@chakra-ui/react'
+import { useState } from 'react'
+import { Layout } from '../../components/layouts/layout'
```

Comment:
Do you mean importing Layout with an absolute path, so that the import line will be the same for everyone? That sounds good. I'm interested.

(I agree that it doesn't need to be so on this PR.)

Reply:
The deeper the file hierarchy, the harder it is to read, like `from ../../../../components/hoge`, so you're trying to make it readable from the root path! 

I checked and it seems that nextjs exists as a feature
https://nextjs.org/docs/advanced-features/module-path-aliases